modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Numeric/Numeric.es.js

```diff
@@ -82,9 +82,19 @@ const Numeric = ({
 		if (inputRef.current) {
 			let newValue = value;
 
+			let decimalSymbol = symbols.decimalSymbol;
+
+			if (
+				newValue &&
+				!newValue.includes('.') &&
+				symbols.decimalSymbol != ','
+			) {
+				decimalSymbol = ',';
+			}
```

Comment:
@alinedoleron, for which languages did you test this code? Would be important we test for languages with different decimal points.

Reply:
I've tested in some languages: English(.), Portuguese(,), Japanese(.), Hungarian(,) and Finnish(,).

Reply:
Awesome!